test/presenters/document_collection_presenter_test.rb

```diff
@@ -124,4 +124,21 @@ class GroupWithDocumentsWhenThereIsNoLinksDocuments < TestCase
       assert_empty group_with_missing_documents
     end
   end
+
+  class GroupWithDocumentsWhenThereAreWithdrawnDocuments < TestCase
+    test "does not present withdrawn documents" do
+      presenter = presented_item(
+        "document_collection_with_withdrawn_links_documents"
+      )
+
+      groups_without_withdrawn_documents = presenter.groups.first["documents"]
```

Comment:
Or possibly just `presented_documents`? The withdrawn bit is redundant as that is in the name of the test.